### ISSUE TITLE
Item Frames and Armor Stands in most buildings pop off immediately

### ISSUE THREAD

**Issue by Paril** [ldtteam/minecolonies#2931]
### Minecolonies version

Version: minecolonies-1.12.2-0.9.3-RELEASE-universal.jar

### Expected behavior
Item Frames are built on walls properly and Armor Stands exist.

### Actual behaviour
In Medieval Dark Oak building set (Deliveryman 4 and Barracks 4 are good tests for this), Item Frames tend to pop off, be empty, or face the wrong orientation and pop off once a block near it gets updated. I find them littering the floor after upgrading in a lot of cases.

Similarly, Barracks requires Armor Stands + armor to build but are nowhere to be found.

**Comment by PoroUsedSnax** [comment]
Unfortunately a well known issue at the moment but currently we're unaware of what could be causing this, when our lead dev Raycoms is off holidays he will be looking into it.

I will note it will likely also affect your placement of anything that can have various rotations (stairs, doors, furnaces, etc) Seems to be just issues for a player though, not the builders.

**Comment by Paril** [comment]
Is this what is causing my stairs, etc to rotate randomly when I place it? Weird that Minecolonies would be doing this. I do have various buildings that I've mirrored/rotating with build tool, is that potentially the cause?

EDIT: Now that I think about it, I've made two colonies so far, one where I never used the Building Tool (just placed the block down and used Build) and one where I used the Building Tool and used mirror/rotate. The latter colony only has this issue.
